src/doc-builders.js

```diff
@@ -46,6 +46,7 @@ function group(contents, opts) {
     type: "group",
     contents: contents,
     break: !!opts.shouldBreak,
+    addedLine: !!opts.addedLine,
```

Comment:
found the comments at https://github.com/Automattic/calypso-prettier/pull/1/files#diff-76f26752c3761490942405c0704a8e1dR357.  Its definitely a level above my knowledge of how prettier works internally.

I'll do my best to try to understand it but won't slow down 🚢  because of it!
I'll do some testing later today